### ISSUE TITLE
Stop schedule app instances from hotlinking to this repo

### ISSUE THREAD

**Issue by mmmavis** [mozilla/mozfest-event-app#304]
[schedule-app-core](https://github.com/mozilla/schedule-app-core) isn't npm controlled and do not have versioning. Hotlinking to files on this repo is a really bad idea. Any file or code changes made here can instantly break live schedule app instances that rely on `schedule-app-core` files/scrips/template/code as dependencies. 

As I was updating files on this repo, I had to also make sure files and code in other schedule app repos (eg., [mofo-allhands](https://github.com/mozilla/mofo-allhands) and [mozfest-schedule-app](https://github.com/mozilla/mozfest-schedule-app/)) are updated accordingly. This increases lots of unneccesary work for devs who are famailiar with **all these repos** and it's nearly impossible for other devs to keep all these repos up and running at all times.

#### [mofo-allhands](https://github.com/mozilla/mofo-allhands)

- [x] Stop hotlinking to files on `schedule-app-core`. This repo should have its own copy of scripts and files. (https://github.com/mozilla/mofo-allhands/issues/43 , DONE)

#### MozFest Schedule App

- [x] Decide which repo we want to keep - `schedule-app-core` or `mozfest-schedule-app`. And delete the other repo. (decision: keep `schedule-app-core`)
- [x] The repo that we are keeping will be used for developing MozFest schedule app. https://github.com/mozilla/schedule-app-core/issues/317

#### [schedule-app-data-processor](https://github.com/mozilla/schedule-app-data-processor)

- [x] this should be renamed to something related to "mofo-allhands" (repo has been renamed to `mofo-allhands-event-app-data-processor`)

#### Create a new repo for MozFest schedule data proccessor script

- [x] Since the spreadsheet we are using for MozFest is different from what we had for `mofo-allhands` we need to create another repo to deal with MozFest spreadsheet data. (Done, https://github.com/mozilla/mozfest-event-app-data-processor)

/cc @xmatthewx @cadecairos @acabunoc

**Comment by mmmavis** [comment]
I'm leaning towards keeping [`schedule-app-core`](https://github.com/mozilla/schedule-app-core) and turning it into the mozfest event app repo. Any objection? 🌞  `schedule-app-core` repo is what we used for last year's MozFest (we renamed the repo to `schedule-app-core`) and therefore it holds records of our code/feature improvements. 

@xmatthewx let's just make the decision soon 👍  as this is blocking other things
